Cargo.toml

```diff
@@ -16,6 +16,11 @@ actix-web = "4"
 config = "0.13.3"
 serde = { version = "1", features = ["derive"] }
 tokio = { version = "1", features = ["macros", "rt-multi-thread"] }
+tracing = "0.1"
+tracing-actix-web = "0.7"
+tracing-bunyan-formatter = "0.3"
+tracing-log = "0.1.3"
+tracing-subscriber = { version = "0.3", features = ["registry", "env-filter"] }
```

Comment:
I know this looks a lot of dependencies for just logging, but they are all needed for good logging. I followed the book mostly for this.